src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -121,6 +123,16 @@ public static function fromDsn(string $dsn, array $redisOptions = [], \Redis $re
         ], $connectionCredentials, $redisOptions, $redis);
     }
 
+    private static function validateOptions(array $amqpOptions): void
```

Comment:
bad name :)

Reply:
Hi,
what should it be in your opinion ?

Reply:
`$amqpOptions` => `$redisOptions` (or `$options`)?

Reply:
oups you're right, i was thinking you were talking about the function's name